### ISSUE TITLE
ol.control.overlay show func fires "change:visible" event two times.

### ISSUE THREAD

**Issue by ergin-duran** [Viglino/ol-ext#328]
Hello,

i was listening "change:visible" events on ol.control.Overlay (menu). When I toggled menu, i see that show func fires this event two times.

I try this to change hamburger icon on navbar.  I toggled icon with menu but while i am spamming on hamburger icon, it sometimes get confused. So i see close icon on navbar but menu has already closed.

Can you check 6051 line in ol-ext (v3.1.4) at ol.control.Overlay.prototype.show..
i commented it for now  because the _timeout func has already called "change:visible" visible=true.

Long story short, there's a problem with show func in ol.control.overlay. When you toggle menu, it fires "change:visible" event two times.

You may already fixed it. But i just met with this issue on v3.1.4 (latest release).

**Comment by Viglino** [comment]
It seems that line 94 in `src\control\Overlay.js` should be removed...
Anyway, you have to look at the `visible` attribute of the event to know if the menu is open/closed.

**Comment by Viglino** [comment]
Done in the last commit
